Bexchange.Domain/DtoModels/LoginUserDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BexchangeAPI.Domain.DtoModels
+{
+    public class LoginUserDTO
```

Comment:
Why do you need that Dto? If you use it to receive data from front end you need to create Request model